app/views/Project/EntryEdit/index.tsx

```diff
@@ -489,6 +562,16 @@ function EntryEdit(props: Props) {
         [setFormError, formValidate, bulkUpdateEntries, projectId, alert, setFormValue],
     );
 
+    const handleFinalizeClick = useCallback(
+        () => {
+            setIsFinalizeClicked(true);
```

Comment:
We also need to clear this when there's local error and no request can be made to the server.